app/src/main/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIClient.java

```diff
@@ -336,10 +338,9 @@ public void onFailure(Call<State> call, Throwable t) {
 
     }
 
-    public void postImg(final Context context, final ProductImage image) {
+    public void postImg(final Context context, final ProductImage image, ImageUploadListener imageUploadListener) {
```

Comment:
Please refactor to remove duplicated code in `postImg(final Context context, final ProductImage image, ImageUploadListener imageUploadListener)` and `postImg(final Context context, final ProductImage image)`

Reply:
Everything else looks good, last item here.